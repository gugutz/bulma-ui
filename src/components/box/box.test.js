import React from 'react'
import renderer from 'react-test-renderer'
import Box from './box'

describe('<Box /> component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Box>text</Box>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
