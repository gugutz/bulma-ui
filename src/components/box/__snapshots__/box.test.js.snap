// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> component should render correctly 1`] = `
<div
  className="box"
>
  text
</div>
`;
